tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -474,9 +474,23 @@ class MklConvOp : public OpKernel {
 
   explicit MklConvOp(OpKernelConstruction* context) : OpKernel(context) {
     OP_REQUIRES_OK(context, context->GetAttr("dilations", &dilations_));
+
+    // Conv and QuantizedConv ops have different padding attributes
+    // ('padding_list' versus 'explicit_paddings). But one and only one
```

Comment:
Nit: Missing an end quote. Also changing quote style to tick.
```suggestion
    // (`padding_list` versus `explicit_paddings`). But one and only one
```